main.go

```diff
@@ -138,6 +139,27 @@ func getConnWithRetry(flags *flagStorage) (c *gcsx.Connection, err error) {
 ////////////////////////////////////////////////////////////////////////
 
 // Mount the file system according to arguments in the supplied context.
+func createStorageHandle(flags *flagStorage) (storageHandle storage.StorageHandle, err error) {
+	var tokenSrc oauth2.TokenSource
+
+	tokenSrc, err = auth.GetTokenSource(context.Background(), flags.KeyFile, flags.TokenUrl, true)
+	if err != nil {
+		err = fmt.Errorf("get token source: %w", err)
+		return
+	}
+
+	var storageClientConfig = storage.StorageClientConfig{DisableHTTP2: true,
```

Comment:
this is wrong.  If you hardcode the values here then what is the purpose of defining them as flags.
We define a parameter as flag, so that user has the feasibility to set whatever value they want. That flag values should be passed to the storageHandle

Reply:
fixed